keyboards/keychron/c1/keyboard.json

```diff
@@ -0,0 +1,259 @@
+{
+    "manufacturer": "Keychron",
+    "keyboard_name": "Keychron C1",
+    "maintainer": "NetUserGet",
+    "bootloader": "sn32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "dip_switch": true
+    },
+    "lto": true,
```

Comment:
LTO should only be enabled if needed - do you **need** it?

Reply:
Im unsure, it was included in the old version of the C1, therefore I included it here.

Reply:
Please check if it builds without it enabled by default

Reply:
I builds fine without it.